app/decorators/offence_decorator.rb

```diff
@@ -23,4 +30,12 @@ def plea_sentence(plea)
       plea: plea.code&.humanize || t('generic.not_available'),
       pleaded_at: plea.pleaded_at&.to_date&.strftime('%d/%m/%Y') || t('generic.not_available'))
   end
+
+  def mode_of_trial_reason_descriptions
+    mode_of_trial_reasons.map { |mode_of_trial| mode_of_trial_reason_description(mode_of_trial) }
+  end
+
+  def mode_of_trial_reason_description(mode_of_trial)
+    mode_of_trial.description || t('generic.not_available')
+  end
```

Comment:
```suggestion
  def mode_of_trial_reason_description(mode_of_trial_reason)
    mode_of_trial_reason.description || t('generic.not_available')
  end
```

Reply:
the above suggestions are because there is a `mode_of_tria`l attribute on offence which we do not want this to be confused with

Reply:
👍  see comment above